source/app/app-service-container.ts

```diff
@@ -34,8 +34,10 @@ import TargetProvider from '@providers/targets'
 import TrayProvider from '@providers/tray'
 import UpdateProvider from '@providers/updates'
 import WindowProvider from '@providers/windows'
-import WorkspaceProvider from '@providers/workspaces'
 import { dialog } from 'electron'
+import { closeSplashScreen, showSplashScreen, updateSplashScreen } from './util/splash-screen'
```

Comment:
The splash screen never really fit in elsewhere. Now the splash screen also indicates booting of other providers.

One NOTE is an interference with the firstStartGuide that the config provider is showing.